src/Infrastructure/Array/interface/ESMF_ArrayGet.cppF90

```diff
@@ -466,7 +467,7 @@ type(ESMF_KeywordEnforcer), optional:: keywordEnforcer ! must use keywords below
       totalLWidthArg, totalUWidthArg, localrc)
     if (ESMF_LogFoundError(localrc, ESMF_ERR_PASSTHRU, &
       ESMF_CONTEXT, rcToReturn=rc)) return
-      
```

Comment:
Probably exclusive white space change (assuming the other line changed in this file is not needed).

Reply:
Also fixed.